libvast/src/system/partition.cpp

```diff
@@ -905,6 +907,8 @@ partition_actor::behavior_type passive_partition(
       return rp;
     },
     [self](atom::erase) -> caf::result<atom::done> {
+      if (!self->state.partition_chunk)
+        return caf::skip;
```

Comment:
Is this worth logging under DEBUG, perhaps?

Reply:
Now we do.